crates/masking/src/strong_secret.rs

```diff
@@ -126,3 +126,32 @@ impl StrongEq for Vec<u8> {
         bool::from(lhs.ct_eq(rhs))
     }
 }
+
+#[cfg(feature = "proto_tonic")]
+impl<T> prost::Message for StrongSecret<T, crate::WithType>
+where
+    T: prost::Message + Default + Clone + ZeroizableSecret,
+{
+    fn encode_raw(&self, buf: &mut impl bytes::BufMut) {
+        self.peek().encode_raw(buf);
+    }
+
+    fn merge_field(
+        &mut self,
+        tag: u32,
+        wire_type: prost::encoding::WireType,
+        buf: &mut impl bytes::Buf,
+        ctx: prost::encoding::DecodeContext,
+    ) -> Result<(), prost::DecodeError> {
+        self.peek_mut().merge_field(tag, wire_type, buf, ctx)?;
+        Ok(())
+    }
+
+    fn encoded_len(&self) -> usize {
+        self.peek().encoded_len()
+    }
+
+    fn clear(&mut self) {
+        self.peek_mut().clear();
```

Comment:
Nit: Either keep the trailing semicolon in all methods returning unit (`()`) (such as `encoded_len()` and `encode_raw()`), or don't add it in any of them.